### PR TITLE
Add/Modify new keywords for Ver.4.6

### PR DESCRIPTION
**Pull request by yasuking0304**
## Add/Modified new keywords for Ver.4.6

I referred to here and picked up the missing basic keywords.

- added version 4.5-4.6 characters
- added version 4.6 drops-boss
- added version 4.4-4.6 drops
- added version 4.6 enemies
- added version 4.6 items
- added version 4.4-4.6 foods
- added version 4.6 locations
- added version 4.6 weapons

### issue #193

### CODE REVIEW

**Comment by xicri** [comment]
Sorry for the delay to respond, please wait a little bit more... 🙏

I quickly checked your commits and this PR includes the commits you made in the last PR.
Since it is very hard to review that, can you rebase it by the main?

```sh
cd path/to/genshin-langdata
git switch main
git fetch upstream
git rebase upstream/main
```

**Comment by yasuking0304** [comment]
Sorry. I ran rebased. If it doesn't work, I'll re-clone it and recreate the PR.

**Comment by xicri** [comment]
Thanks, but still some old commits are remaining.
Can I rebase on my end? You have to pull this branch again after my rebase.

**Comment by xicri** [comment]
@yasuking0304 Fixed.

File: dataset/dictionary/characters.json5 [hunk 1]
```diff
@@ -784,6 +784,14 @@
     notes: "v4.1 期間限定イベント「流れゆく水に詩を紡いで」に登場する純水精霊",
     notesZh: "v4.1 活动「游水酝诗籍」的纯水精灵",
   },
+  {
+    en: "Sir Pouncelot",
+    ja: "橙色騎士",
+    zhCN: "大桔骑士",
+    pronunciationJa: "おれんじきし",
+    tags: [ "mondstadt", "character-sub" ],
```
Comment (xicri) [inline dataset/dictionary/characters.json5:792]: Can you add `"event"` tag and move under events.json5?
I add the event-only characters in event.json5.

File: dataset/dictionary/characters.json5 [hunk 2]
```diff
@@ -4364,6 +4378,51 @@
     zhCN: "纳齐森科鲁兹",
     tags: [ "fontaine", "character-sub" ],
   },
+  {
+    en: "Peruere",
+    ja: "ペルヴェーレ",
+    zhCN: "佩露薇利",
+    tags: [ "fontaine", "character-sub" ],
+    notes: "「アルレッキーノ」の本名",
```
Comment (xicri) [inline dataset/dictionary/characters.json5:4386]: IMO, Peruere should be considered a main character.
I also want to add `enemy-boss`.

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss" ],
    notes: "アルレッキーノの本名",
```

File: dataset/dictionary/characters.json5 [hunk 3]
```diff
@@ -4607,7 +4666,7 @@
     zhCN: "阿蕾奇诺",
     notes: "[召使](/the-knave/)の通称",
     notesZh: "[仆人](/the-knave/)的通称",
-    tags: [ "snezhnaya", "character-sub" ],
+    tags: [ "snezhnaya", "fontaine", "character-main" ],
```
Comment (xicri) [inline dataset/dictionary/characters.json5:4669]: Can you also add `enemy-boss` here?

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss" ],
```

File: dataset/dictionary/characters.json5 [hunk 4]
```diff
@@ -4616,7 +4675,7 @@
     pronunciationJa: "めしつかい",
     notes: "[アルレッキーノ](/arlecchino/)のコードネーム",
     notesZh: "[阿蕾奇诺](/arlecchino/)的代号",
-    tags: [ "snezhnaya", "character-sub", "title" ],
+    tags: [ "snezhnaya", "fontaine", "character-main", "title" ],
```
Comment (xicri) [inline dataset/dictionary/characters.json5:4678]: Can you also add `enemy-boss` here?

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss", "title" ],
```

**Review by xicri — CHANGES_REQUESTED** [review]
@yasuking0304 Thanks!
Please review the comments I added :bow:

**Comment by xicri** [comment]
Since my change requests are not so large, I will commit them by myself.

**Comment by xicri** [comment]
Sorry, something went wrong and the PR is accidentally closed. Fixing...

**Comment by xicri** [comment]
@yasuking0304 I couldn't restore this PR and I opened #195 instead.

Sorry, I accidentally overwrote your main branch with the current main branch of this repository (HEAD: c283672).
However, the changes you've sent in this PR are NOT lost.

**Pull request closed without merging**